website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -5143,3 +5143,282 @@ perUser.apply(ParDo.of(new DoFn<KV<String, ValueT>, OutputT>() {
   }
 }));
 {{< /highlight >}}
+
+## 12 Splittable DoFns {#splittable-dofns}
```

Comment:
Missing period

## 12. Splittable `DoFns` {#splittable-dofns}

Reply:
Also add them for each header. This stopped in the last two sections, but we can keep it consistent with the rest of the programming guide here.